msg/en_US/utils.msg

```diff
@@ -1309,3 +1310,25 @@ valid options:\n\
                                   for flashback; default: outputs only SQL statements for flashback\n\
     --oldest                      flashback is performed in chronological order for SQL statements executed from start time to end time;\n\
                                   default: latest\n
+
+$set 59 MSGCAT_UTIL_SET_MEMMON
+1 Mutually exclusive options cannot be used together.\n
```

Comment:
the help message should describe the exclusiveness of that options. Maybe when describing `-a`?